File: javascript-katas/fizz-buzz.js
const FizzBuzz = (max) => {
  let output = [];

  for (var i = 1; i <= max; i++) {
    switch (0) {
      case i % 3 && i % 5:
        output.push('FizzBuzz');
        break;
      case i % 3:
        output.push('Fizz');
        break;
      case i % 5:
        output.push('Buzz');
        break;
      default:
        output.push(i);
    }
  }
  return output;
}

console.log(
  FizzBuzz(100)
);

File: fizz-buzz.rb
def FizzBuzz(max)
  num = 1
  result = []
  max.times do
    if num % 3 == 0 && num % 5 == 0
      result.push('FizzBuzz')
    elsif num % 3 == 0
      result.push('Fizz')
    elsif num % 5 == 0
      result.push('Buzz')
    else 
      result.push(num)
    end
    num += 1
  end
  return result
end

FizzBuzz(100)
